Makefile

```diff
@@ -55,12 +55,21 @@ install-travis:
 # Example:
 #   make check
 #   make test
-#   make check WHAT=pkg/build TESTFLAGS=-v
-check:
+#   make check WHAT=pkg/docker TESTFLAGS=-v
+check:	verify
```

Comment:
yes, I should use test target instead of $(MAKE) test after moving verify-bash-completion script.